src/integration.js

```diff
@@ -40,10 +40,15 @@ function createIntegration (name) {
    * @api public
    */
 
-  function Integration (settings) {
-    if (!(this instanceof Integration)) return new Integration(settings)
+  function Integration (settings, features) {
```

Comment:
Are we OK with `undefined` as the default value for `features`?

Reply:
Yes, this should be fine since the default is handled on [this line](https://github.com/segmentio/integration/pull/91/files/ba184815ea1c260c035d105c21e568282b7b7997#diff-0e0c28a0622c45ce008fe018cf0601f2R48) so we can also fall back to an empty array on `null` values. (iirc, default parameters only applies to `undefined` values)